src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,30 @@
+import ReactDOM from 'react-dom';
+import styles from './Modal.module.css'
+import { CloseIcon } from '@ya.praktikum/react-developer-burger-ui-components';
+import ModalOverlay from '../ModalOverlay/ModalOverlay'
+
+
+const modalRoot = document.getElementById("react-modals");
+
+
+const Modal = (props, setIsModalOpen) => {
```

Comment:
```suggestion
const Modal = ({props, setIsModalOpen}) => {
```
Где деструктуризация?)